### DISCUSSION TITLE
How to disable the inbuilt Saleor Webhooks via configuration ?

### DISCUSSION THREAD

**Discussion by nigelthomas-mozanta** [saleor/saleor#16812] [Q&A]
Is it possible to disable the Inbuilt Saleor Webhooks via configuration without code change? For eg. when we create a product using productVariantBulkUpdate API, is it possible via configuration to not send webHook events ?

What is the correct approach ?

**Comment by korycins** [comment]
> For eg. when we create a product using productVariantBulkUpdate API, is it possible via configuration to not send webHook events ?

To send a webhook event, you need to have an active app and assigned webhooks to it. Without defining them, no webhook events will be sent. There aren't any webhooks active by default, all of them needs to be defined by you (or by installed app)
